conans/client/build_requires.py

```diff
@@ -87,7 +87,7 @@ def install(self, reference, conanfile, installer):
 
     def _install(self, build_requires_references, build_requires_options, installer):
         # No need current path
-        conanfile = self._loader.load_virtual(build_requires_references, None, scope_options=False,
+        conanfile = self._loader.load_virtual(build_requires_references, scope_options=False,
```

Comment:
Now the virtual does not receive the current dir (installing a reference). I think it has more sense like that, and simplifies the source code.